src/util.cr

```diff
@@ -33,5 +33,10 @@ module Wsman
         log.info("Not removing #{path}, does not exist.")
       end
     end
+
+    def self.set_facl_for_user(user, perm, file, recursive = false)
```

Comment:
Instead of relying on separate ACLs, please use a configurable group (with a default value like `web` for example), and assign the appropriate permissions for the group.
This will be needed at a later step when we implement separate users for each site, and it will be much easier to set permissions for the entire group instead of setting ACLs for each user.